x-pack/plugins/stack_connectors/server/connector_types/slack_api/service.ts

```diff
@@ -119,17 +121,27 @@ export const createExternalService = (
     },
   });
 
-  const getChannels = async (): Promise<ConnectorTypeExecutorResult<unknown>> => {
+  const getChannels = async (): Promise<
+    ConnectorTypeExecutorResult<GetChannelsResponse | void>
```

Comment:
Not a fan of mixing `void` type with another type, since `void` implies that the function shouldn't return anything, but instead called for its side effects. Might be better to specify `undefined` or `null` instead of `void`.

Reply:
The void was already there, I am just showing it.